rclcpp/include/rclcpp/any_subscription_callback.hpp

```diff
@@ -141,7 +143,52 @@ struct AnySubscriptionCallback
   >
   void set(CallbackT callback)
   {
-    unique_ptr_with_info_callback = callback;
+    unique_ptr_with_info_callback_ = callback;
+  }
+
+  void dispatch(
+    std::shared_ptr<MessageT> message, const rmw_message_info_t & message_info) {
```

Comment:
There's an uncrustify error on this line due to the open brace:

http://ci.ros2.org/job/ros2_batch_ci_linux/449/testReport/junit/rclcpp.uncrustify/xunit/include_rclcpp_any_subscription_callback_hpp/

besides that, this PR looks ready.